test/integration/integration_suite_test.go

```diff
@@ -111,17 +110,16 @@ var _ = BeforeSuite(func(done Done) {
 
 	nepheControllerManifests := make(map[string]string)
 	k8sClients = make(map[string]client.Client)
-	clusters = strings.Split(clusterContexts, ",")
-	for _, cluster := range clusters {
-		bytes, err := os.ReadFile(manifest)
-		Expect(err).ToNot(HaveOccurred())
-		nepheControllerManifests[cluster] = string(bytes)
+	cluster := clusterContexts
```

Comment:
can we change this argument to use only one cluster instead of multiple cluster?
i.e. L76
flag.StringVar(&cluster, "cluster, "", "cluster context to use. Default is empty.")

This assignment would not be required.

Reply:
@Anandkumar26 Finally I added this back, but only clusterContext for now.

The reason is because we need to set the `cluster` as the original input(empty string in our case) and "default" back and forth in our case, this will cause an error if we do not have the original input.